tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -379,3 +394,48 @@ Page Should Contain Event Log
         ...    xpath=//div[contains(@class,"alert")]//h4[contains(text(),"${expected_result_text}")]
     END
     Capture Page Screenshot
+
+Select Workbench Number Of GPUs
+    [Documentation]    Selects the container size in the workbench creation page
+    [Arguments]     ${gpus}
+    Wait Until Page Contains Element    ${WORKBENCH_GPU_MENU_BTN_XP}
+    Click Button    ${WORKBENCH_GPU_MENU_BTN_XP}
+    Wait Until Page Contains Element    ${WORKBENCH_GPU_ITEM_BTN_XP}/self::*[text()="${gpus}"]
+    Click Element    ${WORKBENCH_GPU_ITEM_BTN_XP}/self::*[text()="${gpus}"]
+
+Edit GPU Number
+    [Documentation]    Edit a workbench
+    [Arguments]     ${workbench_title}    ${gpus}=1
+    Click Action From Actions Menu    item_title=${workbench_title}    item_type=workbench    action=Edit
+    ${status}=    Run Keyword And Return Status    Select Workbench Number Of GPUs    gpus=${gpus}
+    IF    ${status} == ${FALSE}
+        Click Button    ${GENERIC_CANCEL_BTN_XP}
+        Fail
```

Comment:
can you add  a msg telling why is failing?

Reply:
I thought it was implicit since the error in the logs would tell.

I can write something like `it was not possible to select GPU instances`